apps/annotation/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+SET(deps opencv_core opencv_highgui opencv_imgproc opencv_imgcodecs opencv_videoio)
+ocv_check_dependencies(${deps})
+
+if(NOT OCV_DEPENDENCIES_FOUND)
+   return()
+endif()
+
+project(annotation)
+
+ocv_include_directories("${CMAKE_CURRENT_SOURCE_DIR}" "${OpenCV_SOURCE_DIR}/include/opencv")
+ocv_include_modules(${deps})
+
+set(the_target opencv_annotation)
+
+add_executable(${the_target} opencv_annotation.cpp)
+target_link_libraries(${the_target} ${deps})
```

Comment:
Please use `ocv_add_executable` and `ocv_target_link_libraries` instead (like https://github.com/Itseez/opencv/blob/master/apps/createsamples/CMakeLists.txt).

It is required for OpenCV build with "world" module (it is enabled for distribution package via `-DINSTALL_CREATE_DISTRIB=ON`, see https://github.com/Itseez/opencv/blob/master/CMakeLists.txt#L377).